modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/manager/ExtensionErrorsRegistrant.java

```diff
@@ -93,7 +93,7 @@ protected void onOperation(OperationModel model) {
 
   private ErrorType getErrorType(ErrorModel errorModel) {
     ComponentIdentifier identifier = createIdentifier(errorModel.getType(), errorModel.getNamespace());
-    Optional<ErrorType> optionalError = errorTypeRepository.lookupErrorType(identifier);
+    Optional<ErrorType> optionalError = errorTypeRepository.getErrorType(identifier);
```

Comment:
Modify the addErrorType method of the repository as well to include a check of the internal map.